engine/src/main/java/org/entando/entando/aps/system/init/IInitializerManager.java

```diff
@@ -21,6 +21,8 @@
 public interface IInitializerManager {
 
 	public static enum DatabaseMigrationStrategy {DISABLED, AUTO, GENERATE_SQL};
+    
+    public void init() throws Exception;
```

Comment:
why we need to expose an `init` method ? usually if you need something to do after you should use the `InitializingBean` interface ...